### DISCUSSION TITLE
TIDAL artwork always showing the previous song cover

### DISCUSSION THREAD

**Discussion by E46Fanatic** [BimmerGestalt/AAIdrive#499] [Q&A]
Hello there.
First of all, a huge thanks for your work, this is nothing ahort of being just fantastic. Installed your app a few days ago and have been using it in my G31 and what a diference it makes!

I have one minor (but annoying) problem. So, i have enabled TIDAL in the AAIdrive app and when I play music, the art cover diaplayed is always of the previous song. There is no other other delay, the art changes with each song immediately, it just always shows the previous aong. Can I fix this somehow (or am I doing something wrong)? 

Also one question - I can still see in the apps list Spotify and it shows "Spotify" on my HUD when changing media source even if it plays TIDAL (I have actually disabled Spotify in the AAIdrive app). I suppose this is ok? Spotify is some kind of "main" app that the car can communicate with or something like that? 
Update: I think I have found your answer to this in another thread (in sense that Spotify is the only real app etc.) 

And one more minor question - the AAIdrive says that I have a NBTevo ID5, but I think I have ID6 in the 08/2017 G31, no? 

Would appreciate the help very much and thank you again, indeed.

**Comment by hufman** [comment]
Filed as a bug #502

**Comment by hufman** [comment]
As for the NBTevo version, that comes directly from what the car says it is running. I suspect this value is not updated with later versions of the OS and may merely provide confusion.
